.github/workflows/bun-create-templates.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Publish bun-create templates
+
+on:
+  push:
+    branches:
```

Comment:
This is going to run on every single commit to Bun

it needs to only happen when there are changes to the bun create templates

Reply:
Yeah this was supposed to be a draft PR, my bad. I'll do the easy fix here with `paths`.